pkg/prometheus/statefulset.go

```diff
@@ -21,55 +21,49 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/prometheus-operator/prometheus-operator/pkg/webconfig"
-
-	appsv1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 
-	"github.com/go-kit/log"
-	"github.com/pkg/errors"
-
 	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	"github.com/prometheus-operator/prometheus-operator/pkg/k8sutil"
 	"github.com/prometheus-operator/prometheus-operator/pkg/operator"
 )
 
 const (
-	governingServiceName            = "prometheus-operated"
-	defaultRetention                = "24h"
+	GoverningServiceName            = "prometheus-operated"
 	defaultReplicaExternalLabelName = "prometheus_replica"
-	storageDir                      = "/prometheus"
-	confDir                         = "/etc/prometheus/config"
-	confOutDir                      = "/etc/prometheus/config_out"
-	webConfigDir                    = "/etc/prometheus/web_config"
+	StorageDir                      = "/prometheus"
+	ConfDir                         = "/etc/prometheus/config"
+	ConfOutDir                      = "/etc/prometheus/config_out"
+	WebConfigDir                    = "/etc/prometheus/web_config"
 	tlsAssetsDir                    = "/etc/prometheus/certs"
-	rulesDir                        = "/etc/prometheus/rules"
-	secretsDir                      = "/etc/prometheus/secrets/"
-	configmapsDir                   = "/etc/prometheus/configmaps/"
-	configFilename                  = "prometheus.yaml.gz"
-	configEnvsubstFilename          = "prometheus.env.yaml"
-	sSetInputHashName               = "prometheus-operator-input-hash"
-	defaultPortName                 = "web"
-	defaultQueryLogDirectory        = "/var/log/prometheus"
-	defaultQueryLogVolume           = "query-log-file"
+	//TODO: RulesDir should be moved to the server package, since it is not used by the agent.
+	// It is here at the moment because promcfg uses it, and moving as is will cause import cycle error.
+	RulesDir                 = "/etc/prometheus/rules"
+	secretsDir               = "/etc/prometheus/secrets/"
+	configmapsDir            = "/etc/prometheus/configmaps/"
+	ConfigFilename           = "prometheus.yaml.gz"
+	ConfigEnvsubstFilename   = "prometheus.env.yaml"
+	SSetInputHashName        = "prometheus-operator-input-hash"
```

Comment:
this could be moved to pkg/prometheus/server/statefultset.go as it's only used there?